tests/topotests/lib/common_config.py

```diff
@@ -723,6 +724,11 @@ def start_topology(tgen):
         # Loading empty bgpd.conf file to router, to start the bgp deamon
         router.load_config(TopoRouter.RD_BGP, "{}/{}/bgpd.conf".format(TMPDIR, rname))
 
+        # Loading empty ospf.conf file to router, to start the bgp deamon
```

Comment:
to start the bgp daemon?

Reply:
Additionally why are we starting both bgp and ospf now?  Do all the bgp tests that use this function also now need to start ospf?  That is not good imo

Reply:
Check has been added to start ospfd only when ospf is configured in FRR.
Code has been rebased and PR has been updated.